### ISSUE TITLE
mobile view of cookie bar does not permit closing the bottom bar

### ISSUE THREAD

**Issue by giovanisp** [fairdemocracy/vilfredo-client#129]
from #18 

I use a Samsung S3 mini with Chrome mobile browser. If you see the screenshot, the bottom cookie bar does not show the button to accept and won't go away even if i scroll down the screen (I can't scroll horizontally, naturally). Zoomin in works, zoomin out don't. So the message never goes away.

![image](https://cloud.githubusercontent.com/assets/180794/5200032/19a5edf4-7562-11e4-9d88-f9c32ee1d1ed.png)

**Comment by derekpa** [comment]
Yes, if I have time for this release I'll try and fix it. Actually I do see the same if I'm not in landscape mode, where it is visible.

**Comment by giovanisp** [comment]
I have tweaked the footer text. Please compile LESS and update test, so I can check on mobile.

**Comment by giovanisp** [comment]
postponed to "maybe someday" for smartphne compliance. Works fine on tablets by now.
